.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: Ruff
+
+on: [ push, pull_request ]
+jobs:
+  ruff:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Ruff
+        uses: chartboost/ruff-action@v1
+        with:
+          args: check .
+          config: ruff.toml
```

Comment:
There is a warning in the checks:

> Unexpected input(s) 'config', valid inputs are ['args', 'src', 'version']

Are we sure that the actions supports `config`?